packages/react/src/modal/ModalDialog.tsx

```diff
@@ -41,7 +41,7 @@ export const ModalDialog = ({
 					position: 'relative',
 					margin: '0 auto',
 					minHeight: '100vh',
-					animation: `${animateSlideInUp} ${tokens.transition.duration}  ${tokens.transition.timingFunction}`,
```

Comment:
This has two space characters which is not necessary. I've removed this which is why the snapshot had to be regenerated.